li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -39,6 +39,7 @@
   public static final String VALUE_DESERIALIZER_CLASS_CONFIG = ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG;
   public static final String SEGMENT_DESERIALIZER_CLASS_CONFIG = "segment.deserializer.class";
   public static final String AUDITOR_CLASS_CONFIG = "auditor.class";
+  public static final String ENABLE_PASSTHROUGH_CONFIG = ConsumerConfig.ENABLE_SHALLOW_ITERATOR_CONFIG;
```

Comment:
so where is ENABLE_PASSTHROUGH_CONFIG used?  i see you are using ENABLE_SHALLOW_ITERATOR_CONFIG directly in LiKafkaConsumerImpl

Reply:
yeah, it should not be needed, just need to add the enable.shallow.iterator config to the config constructor.